src/action/list_directory.rs

```diff
@@ -169,81 +177,95 @@ fn get_accesses_time(metadata: &Metadata) -> Option<SystemTime> {
     }
 }
 
+/// Returns the last modification time of provided `Metadata`.
 fn get_modification_time(metadata: &Metadata) -> Option<SystemTime> {
     match metadata.modified() {
-        Ok(atime) => Some(atime),
+        Ok(mtime) => Some(mtime),
         Err(err) => {
             warn!("unable to get last modification time: {}", err);
             None
         }
     }
 }
 
+/// Returns the creation time of provided `Metadata`.
 fn get_creation_time(metadata: &Metadata) -> Option<SystemTime> {
     match metadata.created() {
-        Ok(atime) => Some(atime),
+        Ok(creation_time) => Some(creation_time),
```

Comment:
This is inconsistent (other functions use `atime` and `mtime`), why this one does not follow the pattern?

Reply:
Fixed.